core-services/prow/02_config/openshift/machine-os-images/_prowconfig.yaml

```diff
@@ -35,7 +35,9 @@ tide:
   - excludedBranches:
     - main
     - openshift-4.10
+    - openshift-4.11
     - release-4.10
+    - release-4.11
```

Comment:
This is just one (second) part of my initial comment:

> Looks like we'll need to introduce the GA'd release query in this file (one that requires cherry-pick-approved, backport-risk-assessed) and also add 4.11 branches to the complement query

We still need to add that GA'd release query with 4.10 branches